### PR TITLE
Gives the AI one(1) shell if there are no roundstart cyborgs

### PR DESCRIPTION
**Pull request by Bibby0110**
### Intent of your Pull Request
Bounty.

See title. 

### Why is this good for the game?
Gives AI's, especially traitor AI's, some way to interact with the outside world. They're not as good as a regular borg so this is balanced enough in my opinion

#### Changelog

:cl:  
experimental: If there are no roundstart cyborgs the AI gets a complementary shell! Courtesy of NanoTrasen Data Mining Division
/:cl:

Obligatory Jamie bad x2

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -364,18 +364,42 @@ SUBSYSTEM_DEF(ticker)
...
 /datum/controller/subsystem/ticker/proc/equip_characters()
 	var/captainless=1
+
+	var/no_cyborgs = TRUE
+
 	for(var/mob/dead/new_player/N in GLOB.player_list)
 		var/mob/living/carbon/human/player = N.new_character
 		if(istype(player) && player.mind && player.mind.assigned_role)
 			if(player.mind.assigned_role == "Captain")
 				captainless=0
```
Comment (redmoogle) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
				captainless=FALSE
```

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -364,18 +364,42 @@ SUBSYSTEM_DEF(ticker)
 		CHECK_TICK
 
 
+/**
+  * Equips people that are readied up when the round starts
+  *
+  * In addition, responsible for spawning the cyborg shell if there are no roundstart cyborgs and announcing if there ISN'T a Captain present
+  */
 /datum/controller/subsystem/ticker/proc/equip_characters()
 	var/captainless=1
```
Comment (redmoogle) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
	var/captainless=TRUE
```

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -364,18 +364,42 @@ SUBSYSTEM_DEF(ticker)
...
+
 	for(var/mob/dead/new_player/N in GLOB.player_list)
 		var/mob/living/carbon/human/player = N.new_character
 		if(istype(player) && player.mind && player.mind.assigned_role)
 			if(player.mind.assigned_role == "Captain")
 				captainless=0
+			if(player.mind.assigned_role == "Cyborg")
+				no_cyborgs = FALSE
 			if(player.mind.assigned_role != player.mind.special_role)
 				SSjob.EquipRank(N, player.mind.assigned_role, 0)
```
Comment (redmoogle) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
				SSjob.EquipRank(N, player.mind.assigned_role, FALSE)
```

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 4]
```diff
@@ -1249,3 +1249,7 @@
 		old_ai.connected_robots -= src
 	if(connected_ai)
 		connected_ai.connected_robots |= src
+
```
Comment (JamieD1) [inline code/modules/mob/living/silicon/robot/robot.dm:1252]:
```suggestion
```

File: code/controllers/subsystem/ticker.dm [hunk 5]
```diff
@@ -364,18 +364,42 @@ SUBSYSTEM_DEF(ticker)
...
+/**
+  * Equips people that are readied up when the round starts
+  *
+  * In addition, responsible for spawning the cyborg shell if there are no roundstart cyborgs and announcing if there ISN'T a Captain present
+  */
 /datum/controller/subsystem/ticker/proc/equip_characters()
 	var/captainless=1
+
+	var/no_cyborgs = TRUE
+
```
Comment (JamieD1) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
	var/no_cyborgs = TRUE
```

**Review by redmoogle — CHANGES_REQUESTED** [review]
Just change these over to be less shitcodey and so it's more consistent

File: code/controllers/subsystem/ticker.dm [hunk 6]
```diff
@@ -364,18 +364,42 @@ SUBSYSTEM_DEF(ticker)
 		CHECK_TICK
```
Comment (JamieD1) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**